rpcs3/Emu/RSX/Program/GLSLSnippets/RSXProg/RSXVertexFetch.glsl

```diff
@@ -0,0 +1,209 @@
+R"(
+#ifdef _INTEL_GLSL
+// For intel GPUs which cannot access vectors in indexed mode (driver bug? or glsl version too low?)
+// Note: Tested on Mesa iris with HD 530 and compilant path works fine, may be a bug on Windows proprietary drivers
+void mov(inout uvec4 vector, const in int index, const in uint scalar)
+{
+	switch(index)
+	{
+		case 0: vector.x = scalar; return;
+		case 1: vector.y = scalar; return;
+		case 2: vector.z = scalar; return;
+		case 3: vector.w = scalar; return;

```

Comment:
missing default

Reply:
Index comes from hw and can only be 0-3 (unless you're using CUDA/HIP/etc)